unit-10/main.js

```diff
@@ -0,0 +1,132 @@
+// bài 1: In một bảng nhân của một số bất kỳ với các số từ 1 tới 10 và hiển thị kết quả.
+const tableOfMultipliers = num => {
+  for (let i = 1; i <= 10; i++) {
+    console.log(num * i)
+  }
+}
+tableOfMultipliers(10)
+
+/* bài 2: Hàm nhận tham số là n (số nguyên dương),
+  1 <= n <= 30 In các số chẵn từ 1 đến n Vd n = 6: in ra: 2 4 6 */
+const getEven = num => {
+  if (1 <= num && num <= 30) {
+    for (let i = 1; i <= num; i++) {
+      if (i % 2 !== 0) continue
+      console.log(i)
+    }
+  }
```

Comment:
Line 14, 15: bỏ `continue` đi
Hạn chế viết các condition lồng nhau

```suggestion
  if (num < 1 || num > 30) return;

  for (let i = 1; i <= num; i++) {
    if (i % 2 === 0) {
      console.log(i)
    }
  }
```